lib/t/calls.ex

```diff
@@ -35,19 +38,21 @@ defmodule T.Calls do
       Bot.async_post_message(m)
 
       {:ok, call_id}
-    else
-      # TODO error due to user being on another call
-      {:allowed?, false} -> {:error, "call not allowed"}
-      {:devices, []} -> {:error, "no pushkit devices available"}
-      {:push, false} -> {:error, "all pushes failed"}
     end
   end
 
-  def call_allowed?(caller_id, called_id, reference) do
-    missed?(called_id, caller_id) or matched?(caller_id, called_id) or
-      in_live_mode?(caller_id, called_id, reference)
+  @spec ensure_call_allowed(uuid, uuid, DateTime.t()) :: :ok | {:error, reason :: String.t()}
+  defp ensure_call_allowed(caller_id, called_id, reference) do
+    cond do
+      TWeb.CallTracker.in_call?(called_id) -> {:error, dgettext("calls", "receiver is busy")}
```

Comment:
maybe it makes sense to localise error text on frontend?

Reply:
I'll remove the gettext macros.